src/+otp/Problem.m

```diff
@@ -225,6 +225,7 @@ function validateNewState(obj, newTimeSpan, newY0, newParameters)
                 end
                 
                 options = problem.Rhs.odeset(options);
+                % TODO: Octave does not support odextend
                 sol = odextend(sol, problem.Rhs.F, problem.TimeSpan(end), problem.Y0, options);
```

Comment:
odextend basically just merges the structures. I think we can have octave specific code here.

Reply:
We could implement this, but it might be hard to maintain.  If the solution struct changes in a future release we would need to be able to support multiple formats.

Reply:
I think solution structure will not change, as it is a public API. Matlab would rather depricate all the "odeXX" functions and make new ones, than change their output.

Reply:
I'll give a shot to octave support.  I did notice octave transposed some things for event problems which might be annoying.